### ISSUE TITLE
How to list and document all the keyboard shortcuts

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#7023]
I see there has been some reworking of the keyboard shortcuts recently. The relevant tools are in `utils/keycodes.js`. There are functions for the raw shortcuts and the ones to display in a human readable form, there's a well defined list of modifers, etc. It's a good improvement, well done.

However, users need to know all the available keyboard shortcuts. Sooner or later there will be the need to have a place (internal, external?) to list all the shortcuts. Also, there will be the need to document them somewhere in the handbook.

Even as developers, we don't know how many (and which) keyboard shortcuts are currently available. The only way is to manually look for them through the codebase.

I'd like to propose to consider to make this way easier. Maybe, there should be a centralized place, a configuration file that lists all the shortcuts and can then be imported in the various components when needed? Any feedback and suggestions very welcome.

**Comment by rianrietveld** [comment]
@afercia 
I've talked about this with JonAng of the docs team, and he proposed adding this to the new wp-helphub.com site. We can put the WP manual for users of AT there. If you agree.

**Comment by samikeijonen** [comment]
Agreed.

**Comment by mtias** [comment]
Closing in favor of https://github.com/WordPress/gutenberg/issues/5420
